src/odemis/acq/align/fastem.py

```diff
@@ -39,25 +39,43 @@
         image_rotation,
         image_translation
     )
+    from fastem_calibrations.configure_hw import (
+        get_config_asm,
+        configure_asm
+    )
     fastem_calibrations = True
 except ImportError:
     logging.info("fastem_calibrations package not found")
     fastem_calibrations = False
 
-# TODO does it make sense to make this a list?
-OPTICAL_AUTOFOCUS = autofocus_multiprobe
-SCAN_ROTATION_PREALIGN = scan_rotation_pre_align
-SCAN_AMPLITUDE_PREALIGN = scan_amplitude_pre_align
-DESCAN_GAIN_STATIC = descan_gain
-IMAGE_ROTATION_PREALIGN = image_rotation_pre_align
-IMAGE_TRANSLATION_PREALIGN = image_translation_pre_align
-IMAGE_ROTATION_FINAL = image_rotation
-IMAGE_TRANSLATION_FINAL = image_translation
-
 # The executor is a single object, independent of how many times the module (fastem.py) is loaded.
 _executor = model.CancellableThreadPoolExecutor(max_workers=1)
 
 
+class Calibrations(Enum):
```

Comment:
@pieleric Is there a better way to pass None when the module is not available?

Reply:
I think that you could set explicitly each variable that you import as `None` in the except handler. eg `autofocus_multiprobe = None` ... 
The alternative would be to use the Enum as a standard set of constants (eg, 1, 2, 3...), possibly using `auto()` and use a dict to match constant -> function.

Reply:
I will go for the first one. 
For the second one: Where would the dict go?